x-pack/plugins/infra/server/lib/alerting/log_threshold/log_threshold_executor.ts

```diff
@@ -985,3 +1073,91 @@ export const FIRED_ACTIONS: ActionGroup<'logs.threshold.fired'> = {
     defaultMessage: 'Fired',
   }),
 };
+
+const getContextAggregation = (
+  params: Pick<RuleParams, 'timeSize' | 'timeUnit' | 'groupBy'> & { criteria: CountCriteria }
```

Comment:
Nice, thanks for specifying the type here. Do we need `timeSize` and `timeUnit`, though?

Reply:
No, it's not required. Fixed it.